### PR TITLE
ssshadow requests

### PR DESCRIPTION
**Pull request by RipleyTom**
Adds cpu to log.
Adds [DO NOT USE] to D3D12 selection.

### CODE REVIEW

File: rpcs3/Emu/System.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ void fmt_class_string<video_renderer>::format(std::string& out, u64 arg)
 		case video_renderer::opengl: return "OpenGL";
 		case video_renderer::vulkan: return "Vulkan";
 #ifdef _MSC_VER
-		case video_renderer::dx12: return "D3D12";
+		case video_renderer::dx12: return "D3D12[DO NOT USE]";
```
Comment (Nekotekina) [inline rpcs3/Emu/System.cpp]: I won't give permission to edit this strings

Comment (Zangetsu38) [inline rpcs3/Emu/System.cpp]: why not use ? DX12 is better for few game

Comment (paulsapps) [inline rpcs3/Emu/System.cpp]: If it should never be used then why is it even there? 😂

**Comment by amayra** [comment]
BUT WHY ?

**Comment by AniLeo** [comment]
Why what?

**Pull request merged**